client/state/action-types.js

```diff
@@ -354,6 +354,8 @@ export const IMPORT_IS_SITE_IMPORTABLE_ERROR = 'IMPORT_IS_SITE_IMPORTABLE_ERROR'
 export const IMPORT_IS_SITE_IMPORTABLE_RECEIVE = 'IMPORT_IS_SITE_IMPORTABLE_RECEIVE';
 export const IMPORT_IS_SITE_IMPORTABLE_START_FETCH = 'IMPORT_IS_SITE_IMPORTABLE_START_FETCH';
 export const IMPORTER_NUX_URL_INPUT_SET = 'IMPORTER_NUX_URL_INPUT_SET';
+export const IMPORTER_NUX_CLEAR_FROM_SIGNUP = 'IMPORTER_NUX_CLEAR_FROM_SIGNUP';
```

Comment:
Nit pick: 
`IMPORTER_NUX_FROM_SIGNUP_CLEAR` &  `IMPORTER_NUX_FROM_SIGNUP_SET` would keep the differences in actions more clear when reading RTL.